docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -20,9 +20,9 @@ To complete this guide, you need:
 
 * less than 15 minutes
 * an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
+* `JDK 8 or 11+` installed with `JAVA_HOME` configured appropriately
```

Comment:
I don't think this applies as there is no native image generation for JDK 11 yet

Reply:
Yes, agreed, this is a mistake, it should be JDK 8 for now.

Reply:
Does it need to have backticks?

Reply:
Users can have JAVA_HOME pointing to JDK 11 ++ GRAALVM_HOME pointing to 19.2.0.1.

I understand this is native guide, but limiting to JDK 8 only can be confusing

Reply:
Isn't it more confusing to think Graal is working with JDK 11 when it isn't?

Reply:
Another way from this is to mention just GraalVM here and assume line 27 also covers prerequisites from getting-started-guide

Reply:
With you're getting started guide change you still end up in the same boat, that a developer could be using JDK 11 features.

I think it's safer to explicitly say that native image only applies to JDK 8